lib/util/escapeRegExp.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Escapes RegExp special characters from a string
```

Comment:
Wildcards of "*" seem very reasonable - but general regex syntax, implied or otherwise, is dangerous. eslint uses ` git-ignore` iirc to match patterns, but that (and glob/minimatch) tend to apply to filenames, not strings.